docs/testplan/Smartswitch-test-plan.md

```diff
@@ -645,6 +672,667 @@ root@sonic:/home/cisco# show chassis modules status
  *  Verify number of DPUs from inventory file for the testbed and number of DPUs shown in the cli output.
 
 
+### 1.13 Check Memory on DPU
+
+### Steps
+
+ * Infrastructure support will be provided to choose from following clis or in combinations based on the vendor.
+ * Use command on DPU: `show system-memory` to get memory usage on each of those DPUs
+ * Use command on DPU: `show system-health dpu <DPU_NUM>` to check memory check service status
+ * Use command on DPU: `pdsctl show system --events` to check memory related events triggered.
+   (This is vendor specific event monitoring cli) 
+
+#### Verify in
+ * DPU
+   
+#### Sample Output
+```
+On DPU:
+
+root@sonic:/home/admin# show system-memory
+               total        used        free      shared  buff/cache   available
+Mem:            6266        4198        1509          28         765        2067
+Swap:              0           0           0
+root@sonic:/home/admin# 
+root@sonic:/home/admin# 
+root@MtFuji:/home/cisco# show system-health dpu DPU0
```

Comment:
It may not be related here but interesting why do we need to provide dpu module_name when we are running the command on the DPU?

Reply:
Sorry, this one command on switch.

I changed it.